backend/src/routes.ts

```diff
@@ -7,6 +11,141 @@ router.get('/', (_req, res) => {
   res.send('Hello World')
 })
 
-export {
-  router
-}
\ No newline at end of file
+// Player Routes
+router.get('/player', async (_req, res) => {
+  const players = await findMany<IPlayer>('players')
+  res.json(players)
+})
+
+router.get(`/player/:id`, async (req, res) => {
+  const { id } = req.params
+  const player = await findOne<IPlayer>('players', id)
+
+  res.json(player)
+})
+
+router.post('/player', async (req, res) => {
+  const data: IPlayer = req.body
+
+  if (!isPlayer(data)) {
+    res.status(500).end()
+    return
+  }
+
+  const player = await insert<IPlayer>('player', data)
+
+  res.json(player)
+})
+
+router.put(`/player`, async (req, res) => {
+  const data = req.body
+
+  if (!isPlayer(data)) {
+    res.status(500).end()
+    return
+  }
+
+  const player = await update<IPlayer>('players', data)
+
+  res.json(player)
+})
+
+router.delete('/player/:id', async (req, res) => {
+  const { id } = req.params
+  await remove<IPlayer>('players', id)
+
+  res.status(204).end()
+})
+// End player routes
+
+// License Routes
+router.get('/license', async (_req, res) => {
+  const licenses = await findMany<ILicense>('licenses')
+  res.json(licenses)
+})
+
+router.get(`/license/:id`, async (req, res) => {
+  const { id } = req.params
+  const license = await findOne<ILicense>('licenses', id)
+
+  res.json(license)
+})
+
+router.post('/license', async (req, res) => {
+  const data: ILicense = req.body
+
+  if (!isLicense(data)) {
+    res.status(500).end()
+  }
+
+  const license = await insert<ILicense>('license', data)
```

Comment:
change `const license = await insert<ILicense>('license', data)` to `const license = await insert<ILicense>('licenses', data)`